muesli/web/templates/overview.pt

```diff
@@ -2,18 +2,13 @@
 <metal:main metal:use-macro="templates('Fragments/main.pt').macros['main']">
   <metal:main metal:fill-slot="content">
 
-<h1 class="display-4">Startseite</h1>
-
```

Comment:
I liked it more with the big Heading, why remove it?

Reply:
In my opinion it took a lot of space and added not much information of value, so I removed it.
Still, I have no intentions to defend this change in case someone likes it, so I'm happy to introduce it back in :)

Please stand by while I commit.

Reply:
Done :smile:

Reply:
Woops, forgot to push. Fixed now.